syntax = "proto3";
package job_task;




import "ops_automation_sdk/model/ops_automation/mail_info.proto";
import "ops_automation_sdk/model/ops_automation/job_tasks.proto";

/*
GetJobTasksDetail请求
*/
message GetJobTasksDetailRequest {
    /*
    job Task Id
    */
    string jobTaskId = 1 ;
}

/*
GetJobTasksDetailApi返回
*/
message GetJobTasksDetailResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    ops_automation.JobTasks data = 4 ;
}

